doc/extension_store/README.md

```diff
@@ -0,0 +1,76 @@
+# Proposal for Extension Store Manifest
+
+Example for manifest: [manifest.jsonc](./manifest.jsonc)
+
+## Variable Replacements
+
+* Variables placeholders use the `"My variable 'variableName' has value '${variableName}'"` syntax
+* Naming conventions:
+  * User defined variables are in `lowerCamelCase`
+  * Global constants defined by the system admin:
+    * `BFS_SERVICE`
+    * `BUCKET`
+* User defined variables are defined in the `parameters` section. The Extension Store asks the user to input values for all parameters.
+* Variable placeholders in strings (e.g. BucketFS paths or SQL scripts) are expanded before using them.
+
+## Parameters to Store
+
+The Extension Store must store the following information for an installed VS:
```

Comment:
```suggestion
The Extension management component on the cluster must store the following information for an installed VS:
```
"Store" is usually the opposite site. The binary repository is the basis. It gets a "store" once there are for-pay extensions in there.